docs/content/preview/explore/going-beyond-sql/cluster-aware-drivers.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cluster aware client drivers
+headerTitle: Cluster aware client drivers
+linkTitle: Cluster-aware drivers
+headcontent:
+menu:
+  preview:
+    identifier: cluster-aware-drivers
+    parent: going-beyond-sql
+    weight: 400
+rightNav:
+  hideH3: true
+type: docs
+---
+
+YugabyteDB's innovative approach to distributed database systems extends beyond its core architecture to include intelligent client drivers that are specifically designed to interact with distributed clusters effectively. In this section, we will explore the features and benefits of YugabyteDB's cluster-aware [smart client drivers](../../../drivers-orms/smart-drivers/), highlighting their significance in distributed application development.
+
+## External load balancer
+
+Typically when applications have to connect to a multi-node system, they typically use a load balancer which would accept the multiple connections from the client and distribute them across the different nodes in the system underneath it. This would also be the case for an application connecting to a YugabyteDB cluster using a standard PostgreSQL driver.
+
+![Using an external load balancer](/images/explore/scalability/node-addition-loadbalancer.png)
+
+Although the load balancer hides the cluster from the application, neither the load balancer nor application will have any idea of the topology of the cluster and every time a node is added to the cluster, the load balancer config has to be updated. Another hidden detail is that the load balancer adds a hop for the application to talk to the cluster.
+
+Smart drivers play a crucial role in simplifying the process of connecting to and communicating with a database cluster without the need for an external load balancer.
+
+## Cluster information
+
+When an application connects to a cluster using the smart driver, the driver connects any one node in the connection string. Then it fetches information about all the nodes in the universe using the `yb_servers()` function. This list is refreshed at regular intervals (default: 5 minutes). Subsequently, the driver connects to the least-loaded node before returning the connection to the application.
+
+## Load balancing
+
+Now that the driver has fetched information about all the nodes in the cluster, if load balancing is enabled via the connection string option `load_balance=true`, the driver distributes new connection requests made by the application across the different nodes in the cluster. As the application directly connects to the node there is no extra hop here.
```

Comment:
```suggestion
Now that the driver has fetched information about all the nodes in the cluster, if load balancing is enabled via the connection string option `load_balance=true`, the driver distributes new connection requests made by the application across the different nodes in the cluster. In addition, because the application connects directly to the node, there is no extra hop.
```